src/obr/cli.py

```diff
@@ -56,10 +56,17 @@ def check_cli_operations(
 
 
 def copy_to_archive(
-    repo: Union[Repo, None], use_github_repo: bool, log_file: Path, target_file: Path
+    repo: Union[Repo, None], use_github_repo: bool, src_file: Path, target_file: Path
 ) -> None:
-    logging.info(f"will copy from {log_file} to {target_file}")
-    check_output(["cp", log_file, target_file])
+    """Copies files to archive repo"""
+    # ensure target directory exists()
+    target_path = target_file.parents[0]
+    if not target_path.exists():
+        target_path.mkdir(parents=True)
+    logging.info(f"will copy from {src_file} to {target_file}")
+    if src_file.is_symlink():
+        src_file = os.path.realpath(src_file)
```

Comment:
This is required to not end up with a bunch of invalid symlinks in the data repo

Reply:
Ah, yes. I had not considered the symlinks.